src/admin/admin.controller.ts

```diff
@@ -47,10 +47,11 @@ export class AdminController {
   @Get('/reports')
   async getAllReports(
     @Query('isDeleted', ParseBoolPipe) isDeleted: boolean,
+    @Query('type') type: string,
```

Comment:
Consider making this argument nullable, as there could be scenarios where you want to retrieve all reports without applying a type filter.